packages/polymorphic/README.md

```diff
@@ -83,6 +83,24 @@ export const MyInferredComponent = Polymorphic<
 <MyInferredComponent href="mongodb.design" />; // renders as <a>
 ```
 
+#### With a default `as` value
```

Comment:
Maybe provide a little bit of context as to when/why we'd want to use a default versus not (what value is being provided) -- seeing both examples is confusing without context